chia/util/network.py

```diff
@@ -110,7 +110,15 @@ def is_in_network(peer_host: str, networks: Iterable[Union[IPv4Network, IPv6Netw
 
 
 def is_localhost(peer_host: str) -> bool:
-    return peer_host == "127.0.0.1" or peer_host == "localhost" or peer_host == "::1" or peer_host == "0:0:0:0:0:0:0:1"
+    host_str = peer_host.strip("[]")  # remove IPv6 brackets, our code adds this before making the connection
```

Comment:
Yes, our code doesn't seem to add them in anymore as you mention, I'll update to main and run some tests. If this line is removed, the test parameters will need adjusting

Reply:
removed

Reply:
Should we complain about a now incorrectly formed argument?

Reply:
> Should we complain about a now incorrectly formed argument?

I don't think there is a way we end up with a incorrectly formed argument anymore, do you see a way? I would rather just integrate `is_localhost` into `IPAddress` after https://github.com/Chia-Network/chia-blockchain/pull/14152, then there is definitely no way for a incorrectly formed argument :)

Reply:
Someone could add a new use and not think about the limitation.  Maybe the code would already ail out instead of providing a potentially bogus answer.  And yes, ending up as a method would affect the relevance.